### ISSUE TITLE
Insufficient contrast (according to WCAG)

### ISSUE THREAD

**Issue by hdgarrood** [purescript/pursuit#93]
See: http://webaim.org/resources/contrastchecker/

The following have insufficient contrast, according to WCAG:
- Links: background #FFFFFF, foreground #C4943A
- Keywords in code like "forall", "instance": background #F2F5F9, foreground #A0A0A0

Presumably the links issue also affects the main PureScript website as well.

Unfortunately if you simply darken the link colour until it passes, it becomes a rather horrible brown. So I'm not really sure how to proceed here.

**Comment by naehrstoff** [comment]
I experimented with finding a good color for the links using this [contrast finder tool](http://contrast-finder.tanaguru.com/result.html?foreground=%23C4943A&background=%23FFFFFF&isBackgroundTested=false&ratio=4.5&algo=Rgb).

However, what happens when we increase the contrast of the links, is that they become less distinguishable from the rest of the text, i.e. we fix readability but hurt discoverability 😏

I think having the links in bold at 16px (with a darker color on hover in https://github.com/purescript/pursuit/pull/266) is sufficient for the time being. I solved the issue with the keyword color in https://github.com/purescript/pursuit/pull/266/commits/40ee7cdbb1337b9177b04322833982e8f1abbffd.

A color scheme redesign should take this accessibility issue into account. For the time being, I think this is good enough while allowing us to keep a yellow tone (the most difficult color to get good contrast on!) and I suggest to close the issue.
